src/platform/platform_proprietary.cpp

```diff
@@ -15,19 +15,12 @@
  *
  */
 
-#ifndef MULTIPASS_UNSUPPORTED_REMOTE_EXCEPTION_H
-#define MULTIPASS_UNSUPPORTED_REMOTE_EXCEPTION_H
+#include <multipass/format.h>
+#include <multipass/platform.h>
 
-#include <stdexcept>
+namespace mp = multipass;
 
-namespace multipass
+std::string mp::platform::host_version()
```

Comment:
This file isn't really named appropriately anymore. And, this function is really applicable to the `platform` directory since there is no platform specific code. WDYT

Reply:
Hey Scott, I retraced my steps concerning this and I referred to this in this [commit](https://github.com/canonical/multipass/commit/d6cadea974a7240e2aba4c80a7137269dff6cde7)'s message. I guess it's still correct to refer to macOS and Windows as proprietary platforms and include common implementations here, no? Otherwise I'd have to duplicate the code in both.

Reply:
But it does change based on the platform: on Linux, there is a different implementation (to deal with snaps).

Reply:
I've added some file-level docs, let me know if that looks better.

Reply:
Ah, I see. Makes more sense now.